stanford_profile.profile

```diff
@@ -21,3 +27,50 @@ function stanford_profile_install_tasks(&$install_state) {
 function stanford_profile_final_task(array &$install_state) {
   \Drupal::service('plugin.manager.install_tasks')->runTasks($install_state);
 }
+
+/**
+ * Implements hook_preprocess().
+ */
+function stanford_profile_preprocess(array &$variables, $hook) {
+
+  $variables['su_use_theme_logo'] = '1';
+  $myConfigPage = ConfigPages::config('lockup_settings');
+  if (isset($myConfigPage)) {
+    $variables['su_use_theme_logo'] = $myConfigPage->get('su_use_theme_logo')->value;
+    if ($variables['su_use_theme_logo'] == '0') {
+      $variables['su_path_to_custom_logo'] = $myConfigPage->get('su_path_to_custom_logo')->value;
+    }
+  }
+}
+
+/**
+ * Implements hook_form_FORM_ID_alter().
+ */
+function stanford_profile_form_config_pages_lockup_settings_form_alter(array &$form, FormStateInterface $form_state) {
+
+  $img = '<img src="' . base_path() . drupal_get_path('theme', 'stanford_basic') . '/dist/assets/img/lockup-example.png" />';
+  $rendered_image = render($img);
```

Comment:
This line is not needed as all render is doing here is returning the same string you passed it.

Reply:
ok, i poached that from https://github.com/SU-SWS/stanford_basic/blame/8.x-4.x/theme-settings.php#L38

Reply:
Ah, cool, can you fix that theme up as well. That guy doesn't know what he is doing.